docs/mesos/DCOS.md

```diff
@@ -0,0 +1,134 @@
+      the **mem** options to use 50MB or lower (this includes
+      `mem` and `mem-limit`).
+ - Choose **Review and Install**
+ - Double-check configuration then click **Install**.
+
+You can check the status of etcd by visiting the **Services**
+tab. You can install Calico once etcd appears with a status
+of **`Healthy`** (note that this could take a few minutes).
+
+## Install Calico
+You can easily install Calico from within the DCOS UI.
+
+### Get Calico Package
+You will need to link the Calico package from our git
+repository before you can install it.  To do this:
+
+ - Click the **System** tab
+ - Select **Add Repository** in the upper right
+ - Enter a name such as "Calico" for the package
+ - Paste the following git zip archive as the package link:
+   `https://github.com/robbrockbank/universe/archive/calico-universe.zip`
+
+### Install Calico Package
+You can now install the Calico package from the **Universe**
+tab. To start the install:
+ - Go to the **Universe** tab
+ - Find the Calico package
+ - Click the **Install** button then choose **Advanced
+   Installation** in the pop-up window.
+
+![alt tag](../images/mesos/dcos-calico-package-install.png)
+
+##### At this point you have a few options:
+
+In non-production environments used for testing, 
+you can set each memory limit option to have a
+value of 50MB as seen in the image below. This will
+ensure that smaller tasks with low memory requirements
+will be scheduled immediately. The changes include
+modifying `mem-limit-framework`, `mem-limit-install`,
+`mem-limit-etcd-proxy`, `mem-limit-node`, and
+`mem-limit-libnetwork`.
+
+![alt tag](../images/mesos/dcos-calico-config-changes.png)
+
+You could also speed up the install by increasing
+`max-concurrent-restarts`, allowing Calico to restart
+the Mesos agent and Docker services on more than one
+agent at a time.
+
+##### After configuring the options:
+
+Click **Review and Install**, double-check your
+configuration settings, then choose **Install**.
+
+Calico will now begin installation on your cluster.
+
+### Check Calico Status
+It will take a few minutes for Calico to finish
+installing on your cluster.
+
+Calico runs the following tasks on each agent:
+ - Install Docker multi-host networking
+ - Install Mesos net-modules
+ - Configure an Etcd proxy
+ - Run the `calico-node` Docker image
+ - Run the `calico-libnetwork` Docker image
+
+You can check the status of each process on each agent
+by visiting Calico's status web interface:
+
+ - Go to the **Services** tab
+ - Find Calico in the list of running services
+   (note that it may take a few minutes for Calico
+    to appear).
+ - Once the Calico service is `Healthy`, click the
+   square pop-out icon next to the service name to
+   open the Calico status page in a new tab.
+
+![alt tag](../images/mesos/dcos-calico-open-status.png)
+
+If the install is still in its early stages, the
+webpage may fail to load since the Calico service
```

Comment:
The issue here is more that the install restarts Docker and the Agent - which will cause a restart of the framework when it happens on the same agent.  Should be nothing to worry about and the web page will come back online once the framework is restarted.